build.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
This would probably require editing the RetroArch build script too, but do you think we could embed this into each core file as `info.json` or something?

Reply:
This entire file exists to tell EJS what version of the cores package we're using. It would eventually be updated by the GitHub Action - just didn't want to go to that trouble if the changes weren't accepted.

The idea here is that when we create a new cores release on build repo, a GitHub Action would take the version number from the release and update the "version" attribute. Then when EJS is run, it could read this file (which is in the root of the cores.zip) and display the build version in the about box, and also throw an error if the minimumEJSVersion can't be met.

Reply:
I'll be getting around to attempting to integrate these new json files into EJS in the next few days hopefully. I'll do a similar PR to this one as a proposal.